apps/desktop/src/App.tsx

```
@@ -64,9 +64,32 @@ function App() {
   // Convex hooks for Claude Code sync
   const pendingMobileSessions = useQuery(api.claude.getPendingMobileSessions) || [];
   const createConvexSession = useMutation(api.claude.createClaudeSession);
+  const markMobileSessionProcessed = useMutation(api.claude.markMobileSessionProcessed);
+  
+  // Debug logging for mobile sessions (only when there are sessions to avoid noise)
+  useEffect(() => {
+    if (pendingMobileSessions.length > 0 && isAppInitialized) {
+      console.log('🔍 [MOBILE-SYNC] Detected pending mobile sessions:', pendingMobileSessions.length);
+      console.log('🔍 [MOBILE-SYNC] Mobile sessions data:', JSON.stringify(pendingMobileSessions, null, 2));
+    }
+  }, [pendingMobileSessions, isAppInitialized]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)